openlibrary/plugins/upstream/merge_authors.py

```diff
@@ -275,6 +276,9 @@ def GET(self):
         # sort keys by lowest OL number
         keys = sorted(keys, key=lambda key: int(key[2:-1]))
 
+        # Check if any of the author names are RTL
+        is_rtl_lang = any(is_rtl(key) for key in keys)
```

Comment:
It shouldn’t look for whether the Author name is RTL, but instead whether the specific Work title is RTL. I’m not sure whether this is actually looking at the individual Works and the comment is wrong/misleading, or whether it is actually directly looking at the Author name.

Additionally, if a Work _isn’t_ RTL, it shouldn’t be marked as such. E.g., in my screenshot, “Haifa diary” _shouldn’t_ be marked as RTL, but the other two _should_.